features/step_definitions/pagination_steps.rb

```diff
@@ -22,7 +22,8 @@
       click_link page
     end
   else
-    input = find('input[aria-label="Current page"]')
+    # Pagination controls are rendered twice: at the top and at the bottom of the data table
+    input = find('input[aria-label="Current page"]', match: :first)
```

Comment:
This change is not really needed anymore, it was needed when I tried `config.match = :smart` match mode for Cabybara.

But probably we can leave it here to state that we are aware of this duplication, and have it fixed, if we ever switch to `:smart` mode again. (currently there were too many changes to perform here)